### PR TITLE
more labels

### PR DESCRIPTION
**Pull request by ying-w**
A few more labels but some i'm less confident on

### CODE REVIEW

File: src/api/hooks/useGetANS.ts [hunk 1]
```diff
@@ -150,8 +152,13 @@ export const knownAddresses: Record<string, string> = {
   "0xa4e7455d27731ab857e9701b1e6ed72591132b909fe6e4fd99b66c1d6318d9e8":
     "Coinbase 2",
   "0x834d639b10d20dcb894728aa4b9b572b2ea2d97073b10eacb111f338b20ea5d7": "OKX 1",
+  "0x1e6009ce9d288f3d5031c06ca0b19a334214ead798a0cb38808485bd6d997a43": "OKX dapp",
```
Comment (gregnazario) [inline src/api/hooks/useGetANS.ts:153]: This is OKX Marketplace https://explorer.aptoslabs.com/account/0x1e6009ce9d288f3d5031c06ca0b19a334214ead798a0cb38808485bd6d997a43/modules/code/okx_auction?network=mainnet

File: src/api/hooks/useGetANS.ts [hunk 2]
```diff
@@ -73,6 +73,8 @@ export const knownAddresses: Record<string, string> = {
     "Cellana Rewards",
   "0x1c3206329806286fd2223647c9f9b130e66baeb6d7224a18c1f642ffe48f3b4c":
     "Panora Exchange",
+  "0x1c3266eca38df87026255884192b5979eac24cf0dac7a33cfa7af0d27651fb4c":
```
Comment (gregnazario) [inline src/api/hooks/useGetANS.ts]: Remove

File: src/api/hooks/useGetANS.ts [hunk 3]
```diff
@@ -150,8 +152,13 @@ export const knownAddresses: Record<string, string> = {
   "0xa4e7455d27731ab857e9701b1e6ed72591132b909fe6e4fd99b66c1d6318d9e8":
     "Coinbase 2",
   "0x834d639b10d20dcb894728aa4b9b572b2ea2d97073b10eacb111f338b20ea5d7": "OKX 1",
+  "0x1e6009ce9d288f3d5031c06ca0b19a334214ead798a0cb38808485bd6d997a43": "OKX dapp",
   "0x84b1675891d370d5de8f169031f9c3116d7add256ecf50a4bc71e3135ddba6e0":
     "Bybit 1",
+  "0xfd9192f8ad8dc60c483a884f0fbc8940f5b8618f3cf2bbf91693982b373dfdea":
+    "bitfinex 1",
```
Comment (gregnazario) [inline src/api/hooks/useGetANS.ts:157]:
```suggestion
    "Bitfinex 1",
```

File: src/api/hooks/useGetANS.ts [hunk 4]
```diff
@@ -150,8 +152,13 @@ export const knownAddresses: Record<string, string> = {
...
+  "0xdc7adffa09da5736ce1303f7441f4367fa423617c6822ad2fbc8522d9efd8fa4":
+    "htx 1", // Huobi
```
Comment (gregnazario) [inline src/api/hooks/useGetANS.ts:159]:
```suggestion
    "Htx 1", // Huobi
```

File: src/api/hooks/useGetANS.ts [hunk 5]
```diff
@@ -166,14 +173,20 @@ export const knownAddresses: Record<string, string> = {
...
+    "STAN app",
   // Fungible assets
   "0x357b0b74bc833e95a115ad22604854d6b0fca151cecd94111770e5d6ffc9dc2b": "USDt",
   "0xf73e887a8754f540ee6e1a93bdc6dde2af69fc7ca5de32013e89dd44244473cb":
     "USDt contract",
   // Other
+  "0x7e783b349d3e89cf5931af376ebeadbfab855b3fa239b7ada8f5a92fbea6b387":
+    "Oracle: Pyth",
+  "0x07d7e436f0b2aafde60774efb26ccc432cf881b677aca7faaf2a01879bd19fb8":
+    "Oracle: Switchboard",
```
Comment (gregnazario) [inline src/api/hooks/useGetANS.ts:184]: Interesting, why not put the labels in the same order?

```suggestion
  "0x7e783b349d3e89cf5931af376ebeadbfab855b3fa239b7ada8f5a92fbea6b387":
    "Pyth - Oracle",
  "0x07d7e436f0b2aafde60774efb26ccc432cf881b677aca7faaf2a01879bd19fb8":
    "Switchboard - Oracle",
```

Comment (ying-w) [reply src/api/hooks/useGetANS.ts]: actually going to try and be consistent, I don't label in the name for the other ones so going to not label here either

**Review by gregnazario — APPROVED** [review]

**Pull request merged**